### PR TITLE
Fix '_JSError' object has no attribute '' issue

### PR DESCRIPTION
**Pull request by martijnthe**

### CODE REVIEW

**Comment by cjwilliams** [comment]
+1

**Comment by martijnthe** [comment]
@Katharine OK to merge?

File: __init__.py [hunk 1]
```diff
@@ -171,6 +171,9 @@ def __init__(self, impl):
 _PyV8._JSError._jsclass_RangeError = JSRangeError
 _PyV8._JSError._jsclass_InternalError = JSInternalError
 
+# Work-around for bug here https://github.com/pebble/pyv8/blob/master/src/Exception.cpp#L375-L378
+# that ends up trying to pull an emptry string out of the object:
```
Comment (Katharine) [inline __init__.py]: You misspelt "empty".

**Review by Katharine — APPROVED** [review]
Typo aside, looks good to me. Please make the same change to the pyv8 repo.

**Pull request merged**